src/check_cron_runtime.py

```diff
@@ -24,11 +24,25 @@
 from subprocess import Popen, PIPE, STDOUT
 from argparse import ArgumentParser
 from sys import exit
+from os.path import isfile
 
-# A list of cron names that shall get ignored by this check.
-exclude_list = [
-    'am_stockpile_distribution',
-]
+
+# A list of cron names that shall get ignored by this check will go into here
+exclude_file = '/etc/check_cron_runtime_exclude'
+
+
+def parse_exclude_file():
+    """Parses the exclude file and and adds given list
+
+    Returns a combined list
+    """
+
+    if isfile(exclude_file):
+        with open(exclude_file) as fd:
+            excluded_crons = (
+                [i.rstrip('\n') for i in fd.readlines()]
+            )
```

Comment:
```suggestion
            excluded_crons = (
                [line.rstrip('\n') for line in fd]
            )
```
`fd.readlines()` will read the whole file and put it in memory. Although, I don't believe that we need to worry about that, we also don't need to worry that we have everything in memory. Using `for line in fd` will use `fd` as a generator, having just one line in memory all the time.

Adding `filter` to it would also make it empty line safe:
```suggestion
            excluded_crons = (
                list(filter(None, [line.rstrip('\n') for line in fd]))
            )
```
or sticking to comprehensions and generators:
```suggestion
            excluded_crons = (
                line for line in (l.rstrip('\n') for l in fd) if line
            )
```

Reply:
Hmm TIL filter, thanks.